ftw/news/tests/test_news_listing_block.py

```diff
@@ -264,3 +265,34 @@ def test_has_image_class_is_not_set_on_show_lead_image(self, browser):
             map(lambda news_item: news_item.attrib['class'],
                 browser.css('.news-item .body'))
         )
+
+    @browsing
+    def test_news_listing_block_on_homepage(self, browser):
+        applyProfile(self.portal, 'ftw.news:show-on-homepage')
+
+        newsfolder = create(Builder('news folder'))
+
+        create(Builder('news')
+               .within(newsfolder)
+               .titled(u'News On Homepage')
+               .having(show_on_homepage=True)
+               .having(news_date=datetime(2011, 01, 02, 15, 0, 0)))
```

Comment:
I would remove the prefixing `0`s: they mean that the number is base 8, but actually it should be base 10.
```python
>>> 07
7
>>> 08
  File "<stdin>", line 1
    08
     ^
SyntaxError: invalid token
```
That's why github makes them red 😉